interface/ceed-qfunction.c

```diff
@@ -575,6 +621,10 @@ int CeedQFunctionDestroy(CeedQFunction *qf) {
   }
   ierr = CeedFree(&(*qf)->inputfields); CeedChk(ierr);
   ierr = CeedFree(&(*qf)->outputfields); CeedChk(ierr);
+  // Free ctx if identity
+  if ((*qf)->identity && (*qf)->ctx) {
```

Comment:
```suggestion
  if ((*qf)->identity) {
```
It's safe to `CeedFree`, so may as well keep the conditional simple.